### PR TITLE
Tweak heat damage modifier for lizardpeople

### PR DESCRIPTION
**Pull request by Mooshimi**
## About The Pull Request

image

## Why It's Good For The Game

I think it would be more fair to lizardpeople--since the cold damage modifier is 1.5--to have the heat damage modifier set at .5
(I've also seen a lot of complaints about how lizardpeople have nothing good going for them as a species.)

## Changelog

:cl:
balance: Lizardpeople have slightly increased heat resistance. 
/:cl:

### CODE REVIEW

**Comment by moocowswag** [comment]
Believe it or not, it would be better if lizard people would just more quickly become the temperature of their surroundings.

Heat resistance really does not help as a lizard person when you take a long time to cool down due to no temperature stabilization, this is why heat is still extremely deadly to them.

**Comment by JohnFulpWillard** [comment]
Does this PR have maintainer approval?

**Comment by Mooshimi** [comment]
> Does this PR have maintainer approval?

They'll look at it on here and give their opinion will they not?

**Comment by Mooshimi** [comment]
> Believe it or not, it would be better if lizard people would just more quickly become the temperature of their surroundings (For better or for worse)
> 
> Heat resistance really does not help as a lizard person when you take a long time (several minutes!) to cool down due to no temperature stabilization, this is why heat is still extremely deadly to them despite heat resistance being depicted as an upside.
> 
> Another hidden issue is lizard people, as a species, actually seem like they were supposed to be able to survive in slightly hotter temperatures than most other round start species (2nd to ethereal)
> 
> Basically they have to be hotter to start taking damage from high temperatures.
> 
> But that is only on paper because lizards, again do not have temperature stabilization and other species temperature stabilization allows them to indefinitely survive in hotter temperatures than lizards.
> 
> TLDR this pr doesnt really touch the main issue with lizards and buffing their heat resistance does nothing for them as a species while making them appear op to anyone that doesnt play them.

How in the world does a .67 -> .5 make them look op

**Comment by moocowswag** [comment]
> How in the world does a .67 -> .5 make them look op
 
 person sees lizards
 
 Person sees that they take half heat damage and only 50% more cold damage.
 
Without taking pressure into account (fire suit?) person thinks lizards can survive in plasma fires twice as long as normal people while only dieing 50% faster in space

Person believes that lizards are in a strong spot when that isnt the case.

**Comment by moocowswag** [comment]
> I think it would be more fair to lizardpeople--since the cold damage modifier is 1.5--to have the heat damage modifier set at .5 


This is eating at me because watermelon also did something similar to this.

The modifiers are actually perfectly balanced in their current state, lizards take 33% more heat and 33% more cold.

Decimals become more and more exponentially impactful the lower you go, like a 0.25 burn damage modifier would be balanced by a 4x cold damage modifier, 0.2 = 5,   0.1 = 10 and so on.

**Comment by Colovorat** [comment]
Making 0.5 because there's 1.5 on other side is just... Funny
Not to be rude, but looks like you don't know how math works here.
Plus argument about seeing a lot of complaints that lizards don't have anything on good on them is bad, because that's the goal of maintainers. Races other than human can't be better than human

**Comment by LemonInTheDark** [comment]
> > Does this PR have maintainer approval?
> 
> They'll look at it on here and give their opinion will they not?

You need maintainer approval for species changes, see <https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#banned-content> and all.
Closing this for now, feel free to repr if you get a maint on board.

**Comment by LemonInTheDark** [comment]
that or just have em reopen and dab on me
